app/migrations/Version20250512114846.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Mautic\CampaignBundle\Entity\Event;
+use Mautic\CoreBundle\Doctrine\PreUpAssertionMigration;
+
+final class Version20250512114846 extends PreUpAssertionMigration
+{
+    protected function preUpAssertions(): void
+    {
+        $this->skipAssertion(
+            fn (Schema $schema) => $schema->getTable($this->getPrefixedTableName(Event::TABLE_NAME))
```

Comment:
```suggestion
            fn (Schema $schema) => $schema->getTable($this->getTableName())
```
When it exists, let's use it.

Reply:
Updated it.